open_discussions_api/channels/client_test.py

```diff
@@ -0,0 +1,76 @@
+"""Tests for user client API"""
+import json
+
+import pytest
+
+
+def test_create_channel(api_client):
+    """test a happpy path"""
+    resp = api_client.channels.create(
+        title="new test channel",
+        name="examplechannel",
+        public_description="a good channel for all things",
+        channel_type="public"
+    )
+    assert resp.status_code == 201
+    assert json.loads(resp.request.body) == {
+        "title": "new test channel",
+        "name": "examplechannel",
+        "public_description": "a good channel for all things",
+        "channel_type": "public"
+    }
+    assert resp.json() == {
+        "title": "new test channel",
+        "name": "examplechannel",
+        "public_description": "a good channel for all things",
+        "channel_type": "public"
+    }
+
+
+def test_create_private_channel(api_client):
```

Comment:
isn't this the same test as before with just a different parameter?

Reply:
more or less, yeah. I wanted to test both of those explicitly because we constrain the `channel_type` API to be one of `'public' || 'private'`, so I thought it would be good to add explicit test cases for both of those.

Reply:
ok